### ISSUE TITLE
[Feature]: Master supports the reporting interface of io and cpu load

### ISSUE THREAD

**Issue by pengtianyue025** [cubefs/cubefs#1922]
**Describe the solution you'd like**
- Add reporting interface
- Reporting can consider adding dimensions such as computer rooms and racks
- Support cli query display

**Additional context**
Disk io, cpu usage, etc. are system indicators. Master uses these indicators as important decision-making conditions for load balancing of many applications.

**Comment by NaturalSelect** [comment]
Hello, @bboyCH4 .

Is a separate reporting interface necessary?

Load information seems to be available through the datanode heartbeat.

**Comment by NaturalSelect** [comment]
Another question that bothers me is io load reporting.

What information does the master need to collect from datanode?

**Comment by NaturalSelect** [comment]
The computer rooms and racks information can be passed through `add*Node`.

**Comment by NaturalSelect** [comment]
I'm eager to know the process of moving a machine from one region to another.

Could someone tell me?

And when `update*Node` will be called? 

The information helps me to add the computer room and rack information to master.

**Comment by NaturalSelect** [comment]
```log
# ./build/bin/cfs-cli datanode info 172.16.1.101:17310
[Data node info]
  ID                  : 6
  Address             : 172.16.1.101:17310
  Carry               : 0.13218982116178637
  Allocated ratio     : 0.029416826288887327
  Allocated           : 29.47 GB
  Available           : 921.17 GB
  Total               : 1001.85 GB
  Zone                : default
  IsActive            : Active
  Report time         : 2023-06-07 15:45:54
  Partition count     : 0
  Bad disks           : []
  Persist partitions  : []
  ComputerRoom        : 
  Rack                : 
  CpuUsed             : 0
  IoUsed              : 0
```

```log
# ./build/bin/cfs-cli metanode info 172.16.1.104:17210
[Meta node info]
  ID                  : 5
  Address             : 172.16.1.104:17210
  Carry               : 0.9359605535021892
  Threshold           : 0.75
  MaxMemAvailWeight   : 5.15 GB
  Allocated           : 61.60 MB
  Total               : 5.21 GB
  Zone                : default
  IsActive            : Active
  Report time         : 2023-06-07 15:47:00
  Partition count     : 0
  Persist partitions  : []
  ComputerRoom        : 
  Rack                : 
  CpuUsed             : 0
  IoUsed              : 0
```

**Comment by NaturalSelect** [comment]
Hello, @bboyCH4 I have opened a [pull request](https://github.com/cubefs/cubefs/pull/2097) for it, and I'm waiting for your review.
